### PR TITLE
ZK-4349: Throw an error indicating incorrect usage when using both vf…

### PR DESCRIPTION
**Pull request by plijyfes**
…lex and rows

### CODE REVIEW

File: zktest/src/archive/test2/config.properties [hunk 1]
```diff
@@ -3156,6 +3156,7 @@ F86-ZK-4235.zul=A,E,datefmt,library-property
 ##manually##F90-ZK-4365.zul=A,E,Drawer
 ##manually##F90-ZK-4371.zul=A,E,Toast
 ##zats##F90-ZK-4372.zul=A,E,Notification
+##manually##F90-ZK-4349.zul=A,E,vflex,height,rows
```
Comment (DevChu) [inline zktest/src/archive/test2/config.properties]: remove

Comment (DevChu) [inline zktest/src/archive/test2/config.properties]: remove

**Comment by DevChu** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I just pushed up the changes and merged, enjoy!

**Pull request merged**